ADT/WEB_Server/sftp_oag_client_maytech.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/python
+
+# FTP OAG Script
+# Version 2 - maytech copy
+# ben.baylis@flightregister.net
```

Comment:
please remove references to ben/velvetbug/flight register, it confuses the copyright/ownership of the code